lib/data/external/AwsClient.js

```diff
@@ -327,6 +329,12 @@ class AwsClient {
                   `AWS: ${err.message}`)
                 );
             }
+            if (!completeMpuRes.VersionId) {
```

Comment:
This is from the previous commit?

Reply:
Yeah, if we merge the other pr first and rebase I think it should disappear from the diff
I left the change there because I need some of the other changes to this file for this PR (the `missingVerIdInternalError` const)